larpandoracontent/LArHelpers/LArMCParticleHelper.cc

```diff
@@ -809,6 +809,28 @@ void LArMCParticleHelper::SelectCaloHits(const CaloHitList *const pCaloHitList,
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+bool LArMCParticleHelper::IsDescendentOf(const MCParticle *const mcParticle, const int pdg, const bool isChargeSensitive)
+{
+    const MCParticle* particle = mcParticle;
```

Comment:
Usually `pMCParticle` or just `pParticle` for a pointer variable, and `*` by the variable. Also space after `if` and `while` below, and maybe all on one line for the ternary operator? (Getting subjective here.)